Source/WebCore/platform/graphics/cpu/arm/filters/FEBlendNeonApplier.cpp

```diff
@@ -197,8 +197,8 @@ bool FEBlendNeonApplier::apply(const Filter&, const FilterImageVector& inputs, F
     uint32_t sourceA[2] = { 0, 0 };
     uint32_t sourceBAndDest[2] = { 0, 0 };
 
-    sourceA[0] = reinterpret_cast<uint32_t*>(sourcePixelArrayA->bytes())[0];
-    sourceBAndDest[0] = reinterpret_cast<uint32_t*>(sourcePixelArrayB->bytes())[0];
+    sourceA[0] = reinterpret_cast<uint32_t*>(sourcePixelArrayA->bytes().data())[0];
+    sourceBAndDest[0] = reinterpret_cast<uint32_t*>(sourcePixelArrayB->bytes().data())[0];
```

Comment:
I _think_ this is OK because the result of the expression is just the one byte that gets copied.